src/SFA.DAS.Employer.PR.Web/Views/AddAccounts/ReviewAddAccountsRequest.cshtml

```diff
@@ -61,13 +61,13 @@
                     </legend>
                     <div class="govuk-radios" data-module="govuk-radios">
                         <div class="govuk-radios__item">
-                            <input class="govuk-radios__input" id="acceptAddAccountRequestYes" name="AcceptAddAccountRequest" type="radio" value="true">
+                            <input class="govuk-radios__input" id="acceptAddAccountRequestYes" name="AcceptAddAccountRequest" type="radio" value="true" @(Model.AcceptAddAccountRequest.HasValue && Model.AcceptAddAccountRequest.Value ? "checked" : "")>
```

Comment:
put the 'checked' logic in the viewModel as a string that is empty or "Checked" and add coverage.